apps/dev/nextjs/.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "typescript.tsdk": "../../../node_modules/.pnpm/typescript@4.9.4/node_modules/typescript/lib",
+  "typescript.tsdk": "..\\..\\..\\node_modules\\.pnpm\\typescript@4.9.4\\node_modules\\typescript\\lib",
```

Comment:
Should this change be included in the PR? Looks like a Windows specific change.

Reply:
weird, didn't change this manually - maybe happened automatically when i ran `pnpm i` - changed back now!